deploy/kubernetes/metricbeat-kubernetes.yaml

```
@@ -353,6 +353,11 @@ rules:
   - nodes/stats
   verbs:
   - get
+# Comment out if apiserver metricset is enabled
+#- nonResourceURLs:
+#  - "/metrics"
+#  verbs:
+#  - get
```

This rule seems safe (`get` on a metrics endpoint for metricbeat), should we just add it by default?
```suggestion
- nonResourceURLs:
  - "/metrics"
  verbs:
  - get
```